classroom.rb

```diff
@@ -0,0 +1,17 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
+
+  def add_test_student(student)
+    @students << student
+  end
+end
```

Comment:
Hi, Nice Job so far. Please make sure that When setting the classroom for a student it also adds it to the classroom's students:  You need to create the method that serves as `belongs-to`  side (a student belongs to a classroom). 
- Kindly open this [ LINK](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships%20def%20classroom=(classroom)) and pay attention to `Both ways relationships` section
- **HINT**: The code will be written inside `students` class